pkg/addons/data/DaemonSet.apps/kube-service-catalog/controller-manager.yaml

```diff
@@ -50,6 +50,13 @@ spec:
         - mountPath: /var/run/kubernetes-service-catalog
           name: service-catalog-ssl
           readOnly: true
+        - mountPath: /_data/_out/aad-group-mapping.conf
```

Comment:
Why we mount these into kube-SC controller manager?

Reply:
so we can read the aad creds without loading the whole cs blob